packages/vendure-plugin-invoices/src/ui/helpers.ts

```diff
@@ -0,0 +1,102 @@
+import gql from 'graphql-tag';
+export const GET_ORDER = gql`
+  query GetOrder($id: ID!) {
+    order(id: $id) {
+      id
```

Comment:
Do we really need all these fields? Can't we only fetch invoices?

Reply:
Yes we can